### PR TITLE
[UI] Remove unused npm packages

### PR DESCRIPTION
**Pull request by harupy**
## What changes are proposed in this pull request?

Remove unused npm packages.

## How is this patch tested?

- For dependencies, used [npm-check](https://www.npmjs.com/package/npm-check) to find unused packages, and manually checked the UI.
- For dev dependencies, verified `npm test` and `npm run lint` can be executed without errors.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/server/js/package.json [hunk 1]
```diff
@@ -55,16 +50,10 @@
     "enzyme-adapter-react-16": "1.3.0",
     "eslint": "4.18.2",
     "eslint-config-airbnb-base": "~12.0.0",
-    "eslint-config-standard": "10.2.1",
-    "eslint-import-resolver-webpack": "0.8.4",
-    "eslint-loader": "1.9.0",
     "eslint-plugin-chai-expect": "1.1.1",
     "eslint-plugin-chai-friendly": "0.4.0",
```
Comment (harupy) [inline mlflow/server/js/package.json]: Do we use Chai.js?

**Comment by aarondav** [comment]
@harupy Thanks for the PR, this looks good to me. Would you mind rebasing and rerunning it more recently to make sure these packages are still unused?

**Comment by harupy** [comment]
@aarondav Sorry for the delay. Synced with the master.

**Comment by harupy** [comment]
cc @Zangr

**Review by Zangr — APPROVED** [review]
LGTM, thanks for the cleanup!

**Pull request merged**